src/utils/parsers.js

```diff
@@ -0,0 +1,27 @@
+// 커스텀 구분자 추출
+export function extractCustomDelimiter(input) {
+  const CUSTOM_DELIMITER_REGEX = /^\/\/(.+?)\\n/;
+  const customDelimiterMatch = input.match(CUSTOM_DELIMITER_REGEX);
+  return customDelimiterMatch ? customDelimiterMatch[1] : null;
```

Comment:
```suggestion
  const CUSTOM_DELIMITER_REGEX = /^\/\/(?<customDelimiterMatch>.+?)\\n/;
  const { customDelimiterMatch } = input.match(CUSTOM_DELIMITER_REGEX).groups;
  return customDelimiterMatch ?? null;
```
이런식으로 정규식을 바꾸면 match가 매칭한 정규식에 인덱스로 접근하지 않아도 되서 좋더라구요!:)

Reply:
오 배워갑니당

Reply:
match 메서드를 처음 접해봤는데, 더 좋은 코드가 있었네요. 더 공부해서 기록하겠습니다!
코드까지 작성해주시면서 세세한 피드백 감사드립니다😀